jdaviz/utils.py

```diff
@@ -175,6 +175,21 @@ def bqplot_clear_figure(fig):
     setattr(fig, 'axis_registry', {})
 
 
+def alpha_index(index):
+    """Converts an index to A-Z, AA-ZZ
```

Comment:
Since this might be used by other devs down the road, would be nice to expand a little bit with proper numpydoc and maybe even a short example, so they don't have to dig the test code to find out.

Reply:
Also do we have to worry about something providing negative index? Is that possible?

Reply:
this is why I originally had it nested in the method as its not really intended to be used generally 😜  Negative index would give non-alphabet characters as long as `97-index >= 0` and would otherwise raise an error.  I can catch and raise errors for negative indices if we want to keep this general, but that would never happen in its current use since it is always passed a length as the index.

Reply:
Okay, just want to make sure we'll never do fancy slicing numpy-style.